.rubocop.yml

```diff
@@ -5,6 +5,7 @@ require:
 AllCops:
   Exclude:
     - 'Guardfile'
+    - vendor/**/*
```

Comment:
I didn't had it because since the gem don't have this folder I just left it out, but thanks  for the heads up :)

Reply:
It's a common practice to ignore that like https://github.com/github/gitignore/blob/4488915/Ruby.gitignore#L43, and tbh it's the default of the cop https://github.com/rubocop/rubocop/blob/ca8b38ce3/config/default.yml#L68. Unfortunately `Exclude` seems never inherit that, it just overwrites.

I'm not good at RuboCop, so there will be some modern and good techniques that I do not know.